test/functional/apps/console/_misc_console_behavior.ts

```diff
@@ -156,7 +154,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await PageObjects.console.openConsole();
     });
 
-    describe('customizable font size', () => {
+    describe.skip('customizable font size', () => {
```

Comment:
nit: Can we add a comment above with a link to the issue for the test failure, or at least mention that the test is flaky?